corehq/apps/linked_domain/static/linked_domain/js/domain_links.js

```diff
@@ -80,7 +87,38 @@ hqDefine("linked_domain/js/domain_links", [
         // General data
         self.domain = data.domain;
         self.domain_links = ko.observableArray(_.map(data.linked_domains, DomainLink));
+        self.showRemoteReports = function () {
+            if (data.linkable_ucr) {
+                return data.linkable_ucr.length > 0;
+            }
+            return false;
+        };
+
+        self.isUpstreamDomain = ko.computed(function () {
+            return self.domain_links().length > 0;
+        });
+        // doesn't need to be observable because it is impossible to update the existing page to change this property
+        self.isDownstreamDomain = data.is_downstream_domain;
+
+        self.pullTabActiveStatus = ko.computed(function () {
+            return self.isDownstreamDomain ? "in active" : "";
```

Comment:
how is this `computed`? it looks like `isDownstreamDomain` is not an Observable, so its value is never expected to change. If that's the case `ko.computed` isn't necessary here.

`ko.computed` is useful when its computation references live observables, but if the values never change after initialization, it's best to just use a variable.

Reply:
Sorry I missed this comment initially. Yeah I understand when computed is useful vs when it is unnecessary. I think this came from place of optimism in that a future change could make `isDownstreamDomain` observable, and the computed var would be ready to go. However I shouldn't add unnecessary code and I'm sure a computed var has slight overhead compared to a standard var. Made a [separate PR](https://github.com/dimagi/commcare-hq/pull/30071) to fix this since I already merged this PR.